packages/mobile/src/geth/geth.ts

```diff
@@ -254,16 +255,24 @@ async function writeStaticNodes(nodeDir: string, enodes: string) {
   console.info(`writeStaticNodes enodes are "${enodes}"`)
   const staticNodesFile = getStaticNodesFile(nodeDir)
   await RNFS.mkdir(getFolder(staticNodesFile))
+  await deleteFileIfExists(staticNodesFile)
   await RNFS.writeFile(staticNodesFile, enodes, 'utf8')
 }
 
-async function attemptGethCorruptionFix(geth: any) {
+// async function deleteNodeDbIfExists() {
```

Comment:
Can you add a comment explaining why this is commented out if you want to leave it in